### ISSUE TITLE
Need Horizontal FlatList to hold items and still scrolls for DraxViews/Recievers

### ISSUE THREAD

**Issue by vkaralewich** [nuclearpasta/react-native-drax#164]
Is it possible to basically just replicate [this example](https://snack.expo.io/@lafiosca/react-native-drax---basic-drag-and-drop?platform=ios) except instead of having 4 items to drag, you have lets say like 50 and you need to put them in a horizontal flatlist/scrollview? Right now I have it set up to do that but it doesn't scroll  horizontally if I have Drax enabled because when I try to scroll it activates dragging on one of the items. I would like it to scroll if the gesture is horizontally and activate the draxview if the gesture is vertically. Obviously seems like the DraxScrollView is just for reordering items not really holding a bunch of draxviews.

**Comment by lafiosca** [comment]
Unfortunately that example won't even load for me anymore 😥 The idea behind DraxScrollView was to be able to put DraxViews in a scrollable container and have them still do the drag/drop math correctly when the container is scrolled to different positions. If that behavior is not working, then it's possible this is another regression because the library has not been maintained.

I'm not sure what you mean about scrolling activating the drag on an item though. That shouldn't happen unless someone is touch-dragging, unless you're saying you wrote some custom logic.

**Comment by vkaralewich** [comment]
> Unfortunately that example won't even load for me anymore 😥 The idea behind DraxScrollView was to be able to put DraxViews in a scrollable container and have them still do the drag/drop math correctly when the container is scrolled to different positions. If that behavior is not working, then it's possible this is another regression because the library has not been maintained.
> 
> I'm not sure what you mean about scrolling activating the drag on an item though. That shouldn't happen unless someone is touch-dragging, unless you're saying you wrote some custom logic.

Yeah basically I had it in a ScrollView and a DraxScrollView, and it would not scroll. When I tried to scroll right, it would just pickup the object as if I was going to drag it right. There was no custom logic, so I decided to modify my app so that there is no scroll view, it just expands if they need more options.
